composer.json

```diff
@@ -8,7 +8,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-        "ext-redis": "^5.0.2 || ^6.0",
+        "ext-redis": "^5.3.2 || ^6.0",
```

Comment:
Passing a ssl/tls context to the RedisCluster constructor from the phpredis extension is apparently only introduced in version 5.3.2: https://github.com/phpredis/phpredis/blob/develop/CHANGELOG.md 
I noticed this due to the failing pipeline.
This would of course be a breaking change though.

Reply:
I havent had this kind of change ever but I remember that I did modified some redis range in the past. IMO having a check during CD that all the required extension versions are fulfilled should be a good thing, so I do not have a problem with that change personally, but how is this kind of stuff handled in other OSS.
Do you have experience here @Ocramius?